.azure-pipelines/cd-build-deploy-dotnet.yml

```
@@ -0,0 +1,208 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Build, sign, package and deploy .NET packages for Microsoft.Agents.M365Copilot.Core and Microsoft.Agents.M365Copilot.Beta
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - 'dotnet/src/Microsoft.Agents.M365Copilot.Core/**'
+      - 'dotnet/src/Microsoft.Agents.M365Copilot.Beta/Generated/**'
+      - 'dotnet/src/Microsoft.Agents.M365Copilot.Beta/AgentsM365CopilotBetaServiceClient.cs'
```

```suggestion
      - 'dotnet/src/Microsoft.Agents.M365Copilot.Beta/*'
```
(in case csproj changes only)

---

I've changed the include path patterns to include:
* any updates to a .cs file
* changes to csproj